### PR TITLE
[DOCS] Updated README with deployment/debugging instructions

### PR DESCRIPTION
**Pull request by Nefari0uss**
@RocketChat/android

Closes #ISSUE_NUMBER

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,17 @@ echo "sdk.dir="$ANDROID_HOME > local.properties
...
+
+The following steps are only needed if running via command line. They are not needed if you are building via Android Studio.
+
+Ensure that that ADB recognizes your device with `$ adb devices`.
+
+If a single device exists, install via `$ adb install /path/to/apk.apk`.
+
+Assuming you used Gradle like earlier, the file will be called `module_name-debug.apk` in `project_name/module_name/build/outputs/apk/`.
+
+Alternatively, you can simply run `./gradlew installDebug` to build, deploy, and debug all in a single command.
```
Comment (philipbrito) [inline README.md]: Thanks for your PR @Nefari0uss!

What do you think about how to run that command on the Windows machines? 
We could change it to:

```
Alternatively, you can simply run `./gradlew installDebug` (or `gradlew installDebug ` on Windows OS) to build, deploy, and debug all in a single command.
```

or

```
Alternatively, you can simply run `./gradlew installDebug` to build, deploy, and debug all in a single command.
Note: Run  `gradlew installDebug` on Windows machine.
```
Feel free to change it to cover Windows users. 
[Info](https://developer.android.com/studio/build/building-cmdline.html).

**Review by philipbrito — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Nefari0uss) [reply README.md]: Sure. I can change that once I get home from work.

Comment (leonardoaramaki) [reply README.md]: @Nefari0uss Thanks for your contribution! I spot a typo on line 23, where it says **sent** should be **send**

**Comment by Nefari0uss** [comment]
@filipedelimabrito OK, I've made the requested changes. Please review.

File: README.md [hunk 2]
```diff
@@ -10,16 +10,25 @@
...
 
-./gradlew assembleDebug
-```
+Ensure that that ADB recognizes your device with `$ adb devices`.
+
+If a single device exists, install via `$ adb install /path/to/apk.apk`.
+
+Assuming you used Gradle like earlier, the file will be called `module_name-debug.apk` in `project_name/module_name/build/outputs/apk/`.
 
+Alternatively, you can simply run `$ ./gradlew installDebug` (`> gradlew installDebug` on Windows)to build, deploy, and debug all in a single command.
```
Comment (philipbrito) [inline README.md]: Only a spacing among "`[...] Windows)to [...]`" and we are go!

**Review by philipbrito — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Nefari0uss) [reply README.md]: Done and done.

File: README.md [hunk 3]
```diff
@@ -10,16 +10,25 @@
...
-git clone https://github.com/RocketChat/Rocket.Chat.Android.git
-cd Rocket.Chat.Android
+### How to send APK to device
 
-echo "sdk.dir="$ANDROID_HOME > local.properties
+The following steps are only needed if running via command line. They are not needed if you are building via Android Studio.
 
-./gradlew assembleDebug
-```
+Ensure that that ADB recognizes your device with `$ adb devices`.
```
Comment (philipbrito) [inline README.md]: Could you remove the duplicated "that"?

**Review by philipbrito — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Nefari0uss) [reply README.md]: Well that's embarrassing. Fixed.

**Review by philipbrito — APPROVED** [review]
Thank you so much for your time @Nefari0uss!
Thanks for making this project better!

Feel free to submit others PR's ☺

**Comment by philipbrito** [comment]
@leonardoaramaki Could you approve that? Let's merge this PR!

**Review by leonardoaramaki — APPROVED** [review]
LGTM!

**Pull request merged**